tests/lib/account.js

```diff
@@ -344,13 +344,13 @@ define([
 
       test('#accountLock with no email', function () {
         assert.throws(function () {
-          client.accountLock(null, account.input.password);
+          client.accountLock(null, account.input.password); //eslint-disable-line no-undef
         });
       });
 
       test('#accountLock with no password', function () {
         assert.throws(function () {
-          client.accountLock(account.input.email, null);
+          client.accountLock(account.input.email, null); //eslint-disable-line no-undef
```

Comment:
If we're going to disable the rule here (and on line 347 above), let's make sure to file a separate issue to make sure we fix these tests since they're not testing... anything? (since `account` is `undefined` here &mdash; so I think it'd always throw)